content/source/docs/enterprise/run/api.html.md

```diff
@@ -137,13 +144,22 @@ fi
 CONTENT_DIRECTORY=$1
 ORG_NAME="$(cut -d'/' -f1 <<<"$2")"
 WORKSPACE_NAME="$(cut -d'/' -f2 <<<"$2")"
+
+# 2. Create the File for Upload
+
 UPLOAD_FILE_NAME="./content-$(date +%s).tar.gz"
```

Comment:
I'd leave this alone, but just a note for posterity: since `date +%s` is the granularity used for the filename, if this script is executed multiple times in one second the files will stomp all over each other. ASK ME HOW I KNOW. `tfh` now uses a function called `junonia_randomish_int`.